Makefile

```diff
@@ -1,29 +1,30 @@
 .PHONY: test lint run ftest install dev
 
 PYTHON=python3.10
+ARCH=$(shell uname -m)
 
 bin/python:
 	$(PYTHON) -m venv .
 	bin/pip install --upgrade pip
 
 install: bin/python
-	bin/pip install -r requirements.txt
+	bin/pip install -r requirements/$(ARCH).txt
 
 dev: install
-	bin/pip install -r test-requirements.txt
+	bin/pip install -r requirements/tests.txt
```

Comment:
I wonder what is this used for?

Reply:
you mean the tests requirement? it's all the packages use to run tests, that are not installed in production to *run* the service